eventum/static/js/media/upload.js

```diff
@@ -16,33 +18,36 @@ $(function() {
     setInitialPositions();
 
     /* let the `btn-choose` div act as the file input */
-    $(".btn-choose").click(function () {
+    $(".btn-choose").click(function() {
         $('input[name="image"]').trigger('click');
         return false;
     });
 
     /* let the `btn-upload` div act as the submit button */
-    $(".btn-upload").click(function () {
+    $(".btn-upload").click(function() {
         var formData = new FormData($('.upload-form')[0]);
         $.ajax({
             url: '/admin/media/upload',
             type: 'POST',
-            xhr: function() {   //don't know what this does, but it works!
+            xhr: function() { //don't know what this does, but it works!
                 myXhr = $.ajaxSettings.xhr();
                 return myXhr;
             },
             //Ajax events
             success: function(data, textStatus, jqXHR) {
                 var response = jQuery.parseJSON(jqXHR.responseText);
-                if (response.extension == null && response.filename == null){
-                    $('#images-ajax-loadpoint').load("/admin/media/view", function(response, status){
-                        if (status == "error"){
+                if (response.extension == null && response.filename == null) {
+                    var ajaxPath = "/admin/media/image?mode="
+                    var currMode = document.getElementsByClassName("image-grid")[0].className
```

Comment:
this variable name is not clear, either change the variable name or actually get the mode on this line.